.github/workflows/release.yml

```diff
@@ -1,17 +1,21 @@
 ---
 name: Release
+
 env:
     EXODUS_API_KEY: ${{secrets.EXODUS_API_KEY}}
     KEYSTORE: ${{secrets.RELEASE_KEYSTORE}}
     KEYSTORE_PGP_PASSPHRASE: ${{secrets.RELEASE_KEYSTORE_PGP_PASSPHRASE}}
     KEYSTORE_PASSWORD: ${{secrets.RELEASE_KEYSTORE_PASSPHRASE}}
     KEYSTORE_ALIAS: ${{secrets.RELEASE_KEYSTORE_ALIAS}}
     KEYSTORE_PASSPHRASE: ${{secrets.RELEASE_KEYSTORE_ALIAS_PASSPHRASE}}
+    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
 on:
   push:
     tags:
       - release-v*
   workflow_dispatch:
+
 jobs:
     release:
```

Comment:
This is overindented (4 spaces instead of 2), the whole block should be indented to the left for 2 spaces.